### ISSUE TITLE
Crude XML Tag removal...

### ISSUE THREAD

**Issue by oderwat** [RLovelett/langserver-swift#10]
Popup on all elements seem to work fine for me now!

Maybe just remove the XML Tags from the result by adding a simple filter which translates those tags into either "nothing", "a space" or "a newline". With using "a space" for all unknown tags.

If it is possible to output your own HTML in the popup you could also translate the XML tag into a prefixed css class and all of the tags into div (or span) elements. So one could adjust the output with CSS easily and adding newly encountered tag types would be fairly easy.

**Comment by chrismwendt** [comment]
@RLovelett I'm working on this right now. My approach is to use https://github.com/tadija/AEXML to extract all the text out of the XML.

I'm totally new to Swift and XCode and I ran into an import error (for `import AEXML`) when I build the project in XCode, but it compiles fine with `make debug`. Any tips?

```diff
@@ Package.swift

         .package(url: "https://github.com/RLovelett/swift-package-manager.git", .branch("swift-4.1-branch")),
+        .package(url: "https://github.com/tadija/AEXML.git", .branch("master"))
     ],
...
        .target(
            name: "LanguageServerProtocol",
            dependencies: [
                "BaseProtocol",
                "SourceKitter",
                 "SwiftPM",
+                "AEXML",
             ]

@@ Sources/LanguageServerProtocol/Types/Server.swift:

 import class Workspace.Workspace
+import AEXML // XCode says "No such module 'AEXML'"
 
 ```

**Comment by chrismwendt** [comment]
@vknabel Thanks, that did the trick!

**Comment by chrismwendt** [comment]
Aww yiss! https://github.com/RLovelett/langserver-swift/pull/48

![swift-hovers](https://user-images.githubusercontent.com/1387653/40648252-ccd2f0e0-62e2-11e8-803f-4f9c39374c1c.gif)

**Comment by RLovelett** [comment]
Fixed in 24b35d70bcf627fb17cf45fc270ea7e29ed51e41
